app/controllers/subscriptions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class SubscriptionsController < ApplicationController
+
+    def index
+        render json: { message: 'Hashes of data will get converted to JSON' }
+    end
+
+    def create
+        subscription = Subscription.create(subscription_params)
+        if subscription.save
+            render json: subscription
+        else
+            render json: {error: "Sorry, please try again."}
+        end
+    end
+
+    def destroy
+        subscription = Subscription.find_by email: subscription_params[:email]
+        if subscription
+            if subscription.destroy
+               render json: subscription
+            else
+                render json: {error: "Delete failed."}
+            end
+        else
+            render json: {error: "Cannot find the subscription."}
+        end
```

Comment:
With the bang operator, all this can be eliminated and this will simple suffice:
`@subscription.destroy`, this is possible because all errors will be handled. For example, if the subscription cannot be found, it will raise an ActiveRecord::RecordNotFound error which will be handled my the exception handler module

Special note: Take not of my comment about the before filter action here #3, and take the same approach to set `@subscription`